### PR TITLE
feat(portal): create company list

### PR DESCRIPTION
**Pull request by pccw-reguita**
## Description

* Show list of company
* Create store for customer list (will be removed once api is available)
* Add customer
* Deactivate customer
* Create Tooltip
* Create Modal
* Add svg icons
* Create styled components

## Motivation and Context

Need to create a cutomer company list

## How This Has Been Tested

Test in local

## Screenshots (if appropriate)
image
image
image
image
image
image
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] CI/CD or documentation update (changes to CI/CD pipeline or documentation)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Review by yangpangpangya — DISMISSED** [review]

**Review by p12y — APPROVED** [review]

File: vortex-app/vortex-app-portal/src/pages/Settings/CustomerCompany/__test__/CustomerCompany.test.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    const { getByTestId, getByText } = render(component)
+    const button = getByTestId('add-button')
+    expect(button).toBeInTheDocument()
+    fireEvent.click(button)
+    await waitFor(() => {
+      expect(getByTestId('add-modal')).toBeInTheDocument()
+    })
+    const submitButton = getByText('Cancel')
+    fireEvent.click(submitButton)
+  })
```
[the remark this answers is not in the document]
Comment (pccw-reguita) [reply vortex-app/vortex-app-portal/src/pages/Settings/CustomerCompany/__test__/CustomerCompany.test.tsx:155]: This test was just to make sure that the button is in the DOM as it was under a modal.

File: vortex-app/vortex-app-portal/src/stores/company.store.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      status: 'inactive'
+    },
+    {
+      key: '3',
+      title: 'Reseller customer company C',
+      id: 'IDEO2333',
+      shortName: 'suspendisse',
+      status: 'active'
+    }
+  ],
```
Comment (yangpangpangya) [inline vortex-app/vortex-app-portal/src/stores/company.store.ts:43]: i think we don't need default company list

Comment (pccw-reguita) [reply vortex-app/vortex-app-portal/src/stores/company.store.ts:43]: this store is already deprecated, I will remove it.

**Review by yangpangpangya — APPROVED** [review]

**Review by DaveXiong — APPROVED** [review]
LGTM

**Pull request merged**